dev/run-tests-jenkins.py

```diff
@@ -183,6 +183,8 @@ def main():
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop2.7"
     if "test-hadoop3.2" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop3.2"
+        os.environ["JAVA_HOME"] = "/usr/java/jdk-11.0.1"
+        os.environ["PATH"] = "/usr/java/jdk-11.0.1/bin:%s" % os.environ["PATH"]
```

Comment:
Yes, we need set `PATH` for jdk 11.

Reply:
Let's see if it passes and port this change into your PR #25443 and #25423

Reply:
It works:
![image](https://user-images.githubusercontent.com/5399861/63012224-722a1380-bebc-11e9-8bfa-d7615bc8c4db.png)

Reply:
Nice